scripts/fetch-asyncapi-example.js

```diff
@@ -82,11 +82,11 @@ const buildCLIListFromExamples = async () => {
       };
     } catch (error) {
       // Failed for somereason to parse this spec file, ignore for now
-      console.error(error);
```

Comment:
I would keep this. No reason to remove it IMHO as we want to know if something goes wrong anyway.

Reply:
makes sense